python/tvm/relax/op/unary.py

```diff
@@ -40,23 +192,110 @@ def cos(x: Expr) -> Expr:
     return _ffi_api.cos(x)  # type: ignore
 
 
+def cosh(x: Expr) -> Expr:
+    """Compute element-wise cosh of the input data.
+
+    Parameters
+    ----------
+    x : relax.Expr
+        The input data
+
+    Returns
+    -------
+    result : relax.Expr
+        The computed result.
+
+    Note
+    ----
+    The input tensor is required to have float dtype
+    """
+    return _ffi_api.cosh(x)  # type: ignore
+
+
 def exp(x: Expr) -> Expr:
     """Compute element-wise exp of data.
+
     Parameters
     ----------
     x : relax.Expr
         The input data
+
     Returns
     -------
     result : relax.Expr
         The computed result.
+
     Note
     ----
     The input tensor is required to have float dtype
     """
     return _ffi_api.exp(x)  # type: ignore
 
 
+def floor(x: Expr) -> Expr:
+    """Take floor of input data.
+
+    Parameters
+    ----------
+    x : relax.Expr
+        The input data
+
+    Returns
+    -------
+    result : relax.Expr
+        The computed result.
+    """
+    return _ffi_api.floor(x)  # type: ignore
+
+
+def isfinite(x: Expr) -> Expr:
+    """Check if input value is finite.
+
+    Parameters
+    ----------
+    x : relax.Expr
+        The input data
+
+    Returns
+    -------
+    result : relax.Expr
+        The computed result.
+    """
+    return _ffi_api.isfinite(x)  # type: ignore
+
+
+def isinf(x: Expr) -> Expr:
+    """Check if input value is infinite.
+
+    Parameters
+    ----------
+    x : relax.Expr
+        The input data
+
+    Returns
+    -------
+    result : relax.Expr
+        The computed result.
+    """
+    return _ffi_api.isinf(x)  # type: ignore
+
+
+def isnan(x: Expr) -> Expr:
+    """Check if input value is Nan.
+
+    Parameters
+    ----------
+    x : relax.Expr
+        The input data
+
+    Returns
+    -------
+    result : relax.Expr
+        The computed result.
+    """
+    return _ffi_api.isnan(x)  # type: ignore
```

Comment:
Let’s move these check functions to the bottom of this file with a separating line of comment :-)